platforms/ios/framework/src/TGMapView.mm

```diff
@@ -245,7 +245,7 @@ - (void)setupGestureRecognizers
     [_tapGestureRecognizer requireGestureRecognizerToFail:_doubleTapGestureRecognizer];
 
     _panGestureRecognizer = [[UIPanGestureRecognizer alloc] initWithTarget:self action:@selector(respondToPanGesture:)];
-    _panGestureRecognizer.maximumNumberOfTouches = 1;
+    _panGestureRecognizer.maximumNumberOfTouches = 2;
```

Comment:
Seems like Apple Maps and Google Maps on iOS don't even set a `maximumNumberOfTouches` for panning - that or the maximum is greater than 5 fingers